client/signup/config/flows.js

```diff
@@ -152,6 +152,16 @@ const Flows = {
 		return Flows.filterExcludedSteps( Flows.getABTestFilteredFlow( flowName, flow ) );
 	},
 
+	getNextStepNameInFlow( flowName, currentStepName = '' ) {
+		const flow = Flows.getFlows()[ flowName ];
+		const flowSteps = flow.steps;
+		const currentStepIndex = indexOf( flowSteps, currentStepName );
+		const nextIndex = currentStepIndex + 1;
+		const nextStepName = get( flowSteps, nextIndex, last( flowSteps ) );
+
+		return nextStepName;
+	},
```

Comment:
Any reason to import and use the extra lodash methods here? 

It's not a huge deal, and I totally understand if you're relying on lodash's fallbacks, but the app will fail pretty hard anyway if the flow objects are missing a steps array.

This works,

```
	getNextStepNameInFlow( flowName, currentStepName = '' ) {
		const flowSteps = Flows.getFlows()[ flowName ].steps;
		const nextIndex = flowSteps.indexOf( currentStepName ) + 1;
		return flowSteps[ nextIndex || flowSteps[ flowSteps.length - 1 ] ];
	},
```

Or, if you want to be defensive:

```
getNextStepNameInFlow( flowName, currentStepName = '' ) {
		const flow = Flows.getFlows()[ flowName ] || {};
		if ( flow && Array.isArray( flow.steps ) ) {
			const nextIndex = flow.steps.indexOf( currentStepName ) + 1;
			return flow.steps[ nextIndex || flow.steps[ flow.steps.length - 1 ] ];
		}
		return;
}
```

Please feel free to ignore this comment... :)

Reply:
My thinking is that we could eventually utilise `_.get`s third argument and provide some default here. You're right though, if we're failing to access properties with dot notation, we're already failing elsewhere!

Reply:
In the interests of disclosure, I use `_.get` all the time, and think it would make a nice t-shirt logo.

Reply:
`_.get( 'real' )` 😄